### PR TITLE
Add the Stack overview and File related slices options (defined in th…

### PR DESCRIPTION
**Pull request by oanahogoiu**
Product Library V2 enhancements for "Find related slices", "Stack Overview".

### CODE REVIEW

File: s1tbx-op-insar-ui/src/main/java/org/esa/s1tbx/insar/rcp/actions/InSARStackOverviewActionProductLibraryV2.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+    @Override
+    public void actionPerformed(ActionEvent actionEvent) {
+        RepositoryProduct[] selectedProducts = this.productLibraryToolView.getSelectedProducts();
+        this.productLibraryToolView.readLocalProductsAsync(selectedProducts, this);
+    }
+
+    @Override
+    public void onFailed(Exception exception) {
+        this.productLibraryToolView.showMessageDialog("Stack Overview", "The selected products could be read from the local repository.", JOptionPane.ERROR_MESSAGE);
```
Comment (lveci) [inline s1tbx-op-insar-ui/src/main/java/org/esa/s1tbx/insar/rcp/actions/InSARStackOverviewActionProductLibraryV2.java:51]: I don't understand this error message. Should it say the products could **not** be read?

File: s1tbx-op-insar-ui/pom.xml [hunk 2]
```diff
@@ -91,6 +101,11 @@
             <groupId>org.esa.snap</groupId>
             <artifactId>snap-product-library-ui</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.esa.snap</groupId>
+            <artifactId>snap-product-library-ui-v2</artifactId>
+            <version>${project.version}</version>
+        </dependency>
         <dependency>
```
Comment (lveci) [inline s1tbx-op-insar-ui/pom.xml:109]: duplicate dependency

File: s1tbx-op-insar-ui/src/main/java/org/esa/s1tbx/insar/rcp/actions/InSARStackOverviewActionProductLibraryV2.java (hunk 1 above)
Comment (jeancoravu) [reply s1tbx-op-insar-ui/src/main/java/org/esa/s1tbx/insar/rcp/actions/InSARStackOverviewActionProductLibraryV2.java:51]: The error message should say the products could not be read.
I will add the 'not' word in the message.

File: s1tbx-op-insar-ui/pom.xml (hunk 2 above)
Comment (jeancoravu) [reply s1tbx-op-insar-ui/pom.xml:109]: I opened the pom.xml file and check how many times is defined the dependency searching by 'snap-product-library-ui-v2'.
Only one occurrence exists in the file.

**Pull request merged**